test/one/profiler/test/Runner.java

```diff
@@ -197,6 +199,31 @@ private static void configureSkipTests() {
         }
     }
 
+    private static String formatTestMessage(RunnableTest rt, TestResult result, int i, int testCount, long durationNs) {
+        String duration = String.format("%.1f ms", durationNs / 1e6);
+        String message = result.status().toString() + " [" + i + "/" + testCount + "] " + rt.testInfo() + " took " + duration;
```

Comment:
Can be just a single `String.format` for better readability. Also, calling `toString()` is redundant.